### ISSUE TITLE
Velocity and React Compatibility

### ISSUE THREAD

**Issue by Superpencil** [julianshapiro/velocity#743]
```js
// Something.js

import React from 'react'
import Velocity from 'velocity-animate'
import 'velocity-animate/velocity.ui'
```

```
// Log

Module not found: Error: Can't resolve 'velocity' in '/Users/Username/Repo/node_modules/velocity-animate'
 @ ./~/velocity-animate/velocity.ui.js 19:2-31
 @ ./src/components/Work/Paubox/Interactive.js
 @ ./src/components/Work/Paubox/index.js
 @ ./src/components/index.js
 @ ./src/index.js
 @ multi main
```

Referencing https://github.com/julianshapiro/velocity/pull/479#issuecomment-275056763

**Comment by Rycochet** [comment]
I will look into this when I get time, I'm just very busy right now - I've not forgotten, and have cloned the demonstration repo for when I can do something.

**Comment by abercrave** [comment]
### My setup:

Node v7.7.3
Webpack 2.3.2
Velocity 1.5.0

```javascript
// index.js

import 'velocity-animate';
import 'velocity-animate/velocity.ui';
```

### Error:

```
ERROR in ./~/velocity-animate/velocity.ui.js
Module not found: Error: Can't resolve 'velocity' in '/Users/bevans/projects/musa/frontend/node_modules/velocity-animate'
 @ ./~/velocity-animate/velocity.ui.js 19:2-31
 @ ./app/index.js
 @ multi babel-polyfill ./app/index.js ./app/styles/main.scss
```

**Comment by Superpencil** [comment]
React has switched to an internal dependency manager, haven't tested velocity with it yet

**Comment by abercrave** [comment]
The errors in React and Webpack are identical, but I can create a new issue specific to Webpack 2 if you like.

**Comment by FrancoisH** [comment]
As you're, I weren't able to use import statement...

I have to register it outside react (or vue) like this:
```
window.Velocity = require('velocity-animate');
require('velocity-animate/velocity.ui');
```

**Comment by Rycochet** [comment]
The Velocity namespace for import is now "velocity-animate" and should fix this issue (as of 2.0.2).
